xclim/indices/_threshold.py

```diff
@@ -156,9 +156,11 @@ def cold_spell_days(
     """
     t = convert_units_to(thresh, tas)
     over = tas < t
-    group = over.resample(time=freq)
 
-    out = group.map(rl.windowed_run_count, window=window, dim="time")
+    runs = rl.rle(over)
+    cold_spells = runs.ffill(dim="time") >= window
+    out = cold_spells.resample(time=freq).sum().rename(cold_spell_days)
```

Comment:
```suggestion
    out = cold_spells.resample(time=freq).sum()
```
Not sure what that rename was meant to do, called with a function?

Also, except for units, indices do not set other metadata fields on their output, that is the role of the Indicators.

Reply:
Ah, that's probably left over from running the code locally. I'll commit the suggestion.